docs/kb/semgrep-ci/github-repository-rulesets-semgrep.md

```diff
@@ -0,0 +1,75 @@
+---
+tags:
+  - GitHub
+  - Repository rulesets
+description: Setting up GitHub required workflows for Semgrep scans.
```

Comment:
```suggestion
description: Set up GitHub required workflows to efficiently implement Semgrep scans across many repositories.
```
I haven't been doing a good job catching this, but we use simple present for the description. 

"

(it makes sense when you see the kb cards with the descriptions)